src/main/java/com/google/devtools/build/lib/exec/ExecutionOptions.java

```diff
@@ -122,6 +122,23 @@ public class ExecutionOptions extends OptionsBase {
               + "the 'local' strategy, but reversing the order would run it with 'sandboxed'. ")
   public List<Map.Entry<RegexFilter, List<String>>> strategyByRegexp;
 
+  @Option(
+      name = "allowed_strategies_by_exec_platform",
+      allowMultiple = true,
+      converter = Converters.StringToStringListConverter.class,
```

Comment:
Add a new `LabelToStringListConverter` class (here, not in `Converters`). You can extend [`AssignmentToListOfValuesConverter`](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/common/options/Converters.java?q=symbol%3A%5Cbcom.google.devtools.common.options.Converters.AssignmentToListOfValuesConverter%5Cb%20case%3Ayes) and use a [`LabelConverter`]https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/packages/LabelConverter.java?q=symbol%3A%5Cbcom.google.devtools.build.lib.packages.LabelConverter%5Cb%20case%3Ayes) to handle the key of the conversion.